Source/Core/Logging/FramesLogEvent.swift

```diff
@@ -38,11 +38,13 @@ enum FramesLogEvent: Equatable, PropertyProviding {
     case paymentFormInitialised(environment: Environment)
     case paymentFormPresented
     case paymentFormSubmitted
+    case paymentFormOutcome(token: String)
```

Comment:
Can we rename this paymentFormOutcome to paymentFormSubmittedResult ?

Reply:
✅